### PR TITLE
Jv spring boot book store 3

### PR DESCRIPTION
**Pull request by Vlad1507**

### CODE REVIEW

**Review by okuzan — CHANGES_REQUESTED** [review]
Why so many unrelated changes? Have you merged previously approved PRs into Main?

File: src/main/java/com/store/bookstore/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@SQLRestriction(value = "is_deleted = FALSE")
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, name = "title")
+    private String title;
+    @Column(nullable = false, name = "author")
+    private String author;
```
Comment (Elena-Bruyako) [inline src/main/java/com/store/bookstore/models/Book.java:28]:
```suggestion
    @Column(nullable = false)
    private String title;
    @Column(nullable = false)
    private String author;
```

File: src/main/java/com/store/bookstore/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @Column(nullable = false, unique = true, name = "isbn")
```
Comment (Elena-Bruyako) [inline src/main/java/com/store/bookstore/models/Book.java:29]:
```suggestion
    @Column(nullable = false, unique = true)
```

File: src/main/java/com/store/bookstore/services/impl/BookServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @Override
+    public BookDto update(Long id, CreateBookRequestDto bookRequestDto) {
+        bookRepository.findById(id)
+                .orElseThrow(
+                        () -> new EntityNotFoundException("Can't get a book by id: " + id)
+                );
+        Book book = bookMapper.toModel(bookRequestDto);
+        book.setId(id);
```
Comment (Elena-Bruyako) [inline src/main/java/com/store/bookstore/services/impl/BookServiceImpl.java:53]:
```suggestion
        bookMapper.updateBookFromDto(bookRequestDto, book);
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

There are a lot of changes from previous HW, you should provide implementation of particular HW only

**Review by Elena-Bruyako — APPROVED** [review]

**Pull request merged**